features/deleting.feature

```diff
@@ -0,0 +1,22 @@
+Feature: As a visitor,
+  I would like to delete emails,
+  that are receiven
+
+  Background:
+    Given there is users named:
+      | name      | email            | password |
+      | Test-User | email@random.com | password |
+      | User1     | user1@random.com | password |
+
+  Scenario: Deleting a message
+    Given "Test-User" is logged in
+    And I send an email to "User1"
+    And I should be on the "index" page
```

Comment:
`I should be` feels like an assertion. I'm getting a feeling that you are navigating to the `index` page, right? Please revisit this and rephrase.